gcloud-java-dns/src/test/java/com/google/gcloud/dns/it/ITDnsTest.java

```diff
@@ -845,7 +913,7 @@ public void testListDnsRecords() {
       waitForChangeToComplete(ZONE1.name(), change.id());
       dnsRecordIterator = DNS.listDnsRecords(ZONE1.name(),
           Dns.DnsRecordListOption.dnsName(A_RECORD_ZONE1.name()),
-          Dns.DnsRecordListOption.type(A_RECORD_ZONE1.type()))
+          type(A_RECORD_ZONE1.type()))
```

Comment:
I did not want to do this. IntelliJ went nuts and somehow decided to import it. Fixed.